### PR TITLE
Install the python module from sources

### PR DESCRIPTION
**Pull request by takanokage**
#277

Created installer from sources. (make install after building the sources)
Added scripts for install/uninstall.
Updated the getting started documentation regarding the import of the python module.
Removed `sys.path.append("../..")` from the python examples.

### CODE REVIEW

**Comment by syncle** [comment]
Oh wait, based on the discussion we had, I thought `make install` does not work on windows. However, this PR may break running python tutorials on windows. Is that true?

File: src/Visualization/CMakeLists.txt [hunk 1]
```diff
@@ -53,10 +53,10 @@ else ()
...
 endif ()
 
-install(TARGETS Visualization
-	RUNTIME DESTINATION ${CMAKE_INSTALL_PREFIX}/bin
-	LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/lib
-	ARCHIVE DESTINATION ${CMAKE_INSTALL_PREFIX}/lib
-)
-file(GLOB headers "${CMAKE_CURRENT_SOURCE_DIR}/*.h")
-install(FILES ${headers} DESTINATION ${CMAKE_INSTALL_PREFIX}/include)
+# install(TARGETS Visualization
```
Comment (syncle) [inline src/Visualization/CMakeLists.txt]: please remove commented lines. Applied to other cases too.

File: docs/getting_started.rst [hunk 2]
```diff
@@ -71,15 +71,27 @@ Running Open3D tutorials
...
+    # uninstall the Python module
+    ./uninstall.sh
+
+For quick sanity check, try importing the library from the python interactive shell:
+
+.. code-block:: sh
+
+    python
+
+    >>> import py3d
```
Comment (syncle) [inline docs/getting_started.rst]: Please add more description for windows. Please check my comments in this PR.

File: util/scripts/install.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+echo
+echo installing...
```
Comment (syncle) [inline util/scripts/install.sh]: Can you echo a path where the libraries are installed?

File: util/scripts/uninstall.sh [hunk 4]
```diff
@@ -0,0 +1,8 @@
+echo
+echo uninstalling...
```
Comment (syncle) [inline util/scripts/uninstall.sh]: Can you echo the uninstallation path?

**Comment by takanokage** [comment]
ok, so it works on Windows as well? I didn't try it because my Windows environment is still not working after half a day of attempts. I'm pretty mad right now at how much trouble this is behind the corp network/proxies.

File: util/scripts/install.sh (hunk 3 above)
Comment (takanokage) [reply util/scripts/install.sh]: that is echoed a the end of the install:

```
Install the project...
-- Install configuration: "Release"
-- Installing: /home/dpetre/.local/lib/python3.5/site-packages/py3d.cpython-35m-x86_64-linux-gnu.so
```

File: util/scripts/uninstall.sh (hunk 4 above)
Comment (takanokage) [reply util/scripts/uninstall.sh]: I just made a commit about it.

File: src/Visualization/CMakeLists.txt (hunk 1 above)
Comment (takanokage) [reply src/Visualization/CMakeLists.txt]: fine with me. that was the "install" script for the c/c++ side of things. it's not working so I decided to hide it entirely. removing the code it's also fine.

Comment (takanokage) [reply src/Visualization/CMakeLists.txt]: DONE

File: docs/getting_started.rst (hunk 2 above)
Comment (takanokage) [reply docs/getting_started.rst]: OK, I see, it works by default on Windows too. Awesome :).

Comment (takanokage) [reply docs/getting_started.rst]: DONE

File: util/scripts/uninstall.sh (hunk 4 above)
Comment (takanokage) [reply util/scripts/uninstall.sh]: DONE

**Comment by syncle** [comment]
and please remove 
```
import sys
sys.path.append("../..")
```
in documentations and tutorial scripts if necessary. 
Some scripts will not require `sys`.

**Comment by takanokage** [comment]
I don't know if it overwrites or not. I see the install process detects the existing installation and says:
`-- Up-to-date: /home/dpetre/.local/lib/python3.5/site-packages/py3d.cpython-35m-x86_64-linux-gnu.so`.

**Pull request merged**